locate/src/views/components/Map/LocateForm.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from "react";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import { CardActions, Divider } from "@material-ui/core";
+import { withStyles } from "@material-ui/styles";
+import PropTypes from "prop-types";
+import axios from "axios";
+import { render } from "enzyme";
+
+const styles = theme => ({
+  nested: {
+    paddingLeft: theme.spacing(4)
+  },
+  root: {
+    backgroundColor: theme.palette.background.paper,
+    height: "auto",
+    marginLeft: "1em"
+  }
+});
+
+class LocateForm extends React.Component {
+  // Master location form controls
+  constructor(props) {
+    super(props);
+    this.state = {
+      numberOfDevices: "",
+      mustHaveCoord: "",
+      btnSubmit: false
+    };
+    this.changeHandler = this.changeHandler.bind(this);
+    this.submitHandler = this.submitHandler.bind(this);
+  }
+
+  changeHandler = e => {
+    this.setState({ [e.target.name]: e.target.value });
+    // toggle submit button ON and OFF
+    if (e.target.name == "numberOfDevices") {
+      if (e.target.value != "") {
+        this.setState({ btnSubmit: true });
+      } else {
+        this.setState({ btnSubmit: false });
+      }
+    }
+  };
+
+  submitHandler = e => {
+    e.preventDefault();
+    //functionality after submitting form.
+    // make api call
+    axios
+      .post(
+        `http://localhost:4000/api/v1/map/parishes`,
```

Comment:
This shows that you are running on localhost. Could we use a deployed backend endpoint? 
Also, could we be able to add these links to `locate/src/config/constants.js`?